include/z64player.h

```diff
@@ -318,6 +392,9 @@ typedef enum PlayerFacialExpression {
     /* 15 */ PLAYER_FACE_15
 } PlayerFacialExpression;
 
+// TODO: Contains more than just the face info, should be renamed appropriately
+#define GET_FACE_FROM_JOINTTABLE(jointTable) (((PlayerAnimationFrame*)(jointTable))->faceInfo)
```

Comment:
Does this name come from OoT? I think it's just the eye and mouth indices that are appended onto the jointTable in OoT right?

So in MM, it seems to also store indices related to the left and right hand. So `GET_FACE_HAND_FROM_JOINTTABLE` and `faceHandInfo`?

Reply:
No, this is a name I came up with (before knowing this member has more stuff than just the face information).

I'm not sure about naming it `faceHandInfo`, I would kinda prefer a name that could also apply to OoT. What about `expressionInfo`? (it doesn't need to be suffixed with `Info` btw)

Reply:
`expression` doesn't really capture that it contains info about the hands though. Although I don't really have a good alternative

Reply:
`gesture`, `posture`? I can't really think on a better word :s

Reply:
`ges·ture: a movement of part of the body, especially a hand or the head, to express an idea or meaning.`
That might work for me?

Reply:
That's too much the other way, gestures don't usually include facial expressions. What it's really for is to change parts of the model, be they hands or face parts?

Reply:
It changes hands, eyes and mouth. Do you have a better name?

Reply:
I think we should base it on the idea of changing appearance instead of posture, as it were.

Reply:
`appearanceInfo` :thonk: